src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/jpa/specification/VisitSpecification.kt

```diff
@@ -30,7 +30,7 @@ class VisitSpecification(private val filter: VisitFilter) : Specification<Visit>
     }
 
     filter.endDateTime?.run {
-      predicates.add(criteriaBuilder.lessThanOrEqualTo(root.get(Visit::visitStart.name), this))
+      predicates.add(criteriaBuilder.lessThan(root.get(Visit::visitStart.name), this))
```

Comment:
was that the bug fix

Reply:
nice

Reply:
It fixes the issue with the sequential slots - it's one of the ones in our list to re-visit.